spec/support/lib/module_validation.rb

```diff
@@ -96,6 +100,48 @@ def initialize(mod)
       OVE
     ]
 
+    #
+    # Acceptable architectures
+    #
+    VALID_ARCH_VALUES = [
```

Comment:
An alterantive was to generate this programtically, but that would allow values that we don't want

```
>> Rex::Arch.constants.map { |const| [const, Rex::Arch.const_get(const)] }
=> 
[[:ENDIAN_BIG, 1],
 [:ARCH_LOONGARCH64, "loongarch64"],
 [:IS_ENDIAN_LITTLE, true],
 [:IS_ENDIAN_BIG, false],
 [:X86, Rex::Arch::X86],
 [:Sparc, Rex::Arch::Sparc],
 [:ZARCH, Rex::Arch::ZARCH],
 [:ARCH_ANY, "_any_"],
 [:ARCH_ALL,
  ["x86",
   "x86_64",
   "x64",
   "mips",
   "mipsle",
   "mipsbe",
   "mips64",
   "mips64le",
```